virt/virt-2-5-release-notes.adoc

```diff
@@ -77,6 +77,14 @@ Other operating system templates shipped with {VirtProductName} are not supporte
 //Placeholder for new content.
 
 //CNV-7315 - Expose next run configuration in the UI
+If the virtual machine is running, changes made to the following fields on the *Virtual Machine Overview* page in the web console will not take effect until you restart the virtual machine:
```

Comment:
The fields listed below are not on *Virtual Machine Overview* page.

Reply:
@gouyang The tabs listed above are part of the Workloads > Virtualization > Virtual Machines > Virtual Machine Overview screen. Are you proposing changing 'page' to 'screen'?

Reply:
I would suggest remove `on the *Virtual Machine Overview* page in the web console`.

Reply:
I have removed the reference to the Virtual Machine Overview page.